WordPress/gutenberg
# Editor: Shim meta attributes for early block registrations

Fixes #20494 

This pull request seeks to resolve an issue where blocks which are registered early and use the (now-deprecated) meta attribute source would fail to apply the compatibility shims introduced between #17153 and #20494.

**Testing Instructions:**

There are two ways to test:

Per #20494, following the [meta block tutorial](https://developer.wordpress.org/block-editor/tutorials/metabox/meta-block-1-intro/) should yield a plugin with code such as in the following gist:

https://gist.github.com/aduth/9953fdd7676ce080ab24c2c8a85ae783

From the page, you may be able to <a href="https://gist.github.com/aduth/9953fdd7676ce080ab24c2c8a85ae783/archive/9efb51b8e2887840c8d2cd9454ad4989b105d81c.zip"><img alt="Download zip" valign="middle" src="https://user-images.githubusercontent.com/1779930/75583291-3f500300-5a3b-11ea-84a5-245acb4e7f93.png"></a> to download a ZIP to use as plugin.

Otherwise, you can use the now-updated "Meta Attribute Block" Gutenberg test plugin to verify the behavior. You can do this as a user if you are using the [default development environment](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/getting-started.md) by activating the plugin from the Plugins screen.

You can ensure the end-to-end tests pass:

```
npm run test-e2e packages/e2e-tests/specs/editor/plugins/meta-attribute-block.test.js
```
